### ISSUE TITLE
Return Video Width and Height

### ISSUE THREAD

**Issue by Matheus-Dev** [tcking/GPlayer#1]
Hello, it would be possible to implement a method that returns the height and width of the video, because video in 4K, resolution of 3840 × 2160 pixels, the smartphone closes the application. So I can check if it will support the video before the crash happens.
Thank you very much.

**Comment by tcking** [comment]
yes,you can add you logic to ’gplayer.dart/_onGetPlayerData‘ and find 'onVideoSizeChanged'
